build.gradle

```diff
@@ -140,8 +140,14 @@ subprojects { project ->
     targetCompatibility = JavaVersion.VERSION_1_8
   }
 
-  if (project.name != 'wire-runtime' && project.name != 'japicmp') {
+  if (project.name != 'wire-runtime' && project.name != 'japicmp' && project.name != 'wire-tests') {
```

Comment:
Do we configure publish elsewhere?

Reply:
No. It confuses me where the `uploadArchives` task is coming from, still investigating. Hopefully this is a temporary solution.